src/app/(privateProfile)/analytics/page.tsx

```diff
@@ -1,8 +1,349 @@
+	const [displayMode, setDisplayMode] = useState("rub");
+	const isEmptyPage = false;
+
+	useEffect(() => {
+		const handleResize = () => {
+			setItemsToShow(window.innerWidth <= WINDOW_SIZE ? MIN_ROW : MAX_ROW);
+		};
+
+		handleResize();
+
+		window.addEventListener("resize", handleResize);
+
+		return () => {
+			window.removeEventListener("resize", handleResize);
+		};
+	}, []);
+
+	const handleDisplayChange = (event: ChangeEvent<HTMLInputElement>) => {
+		setDisplayMode(event.target.value);
+	};
+
+	const data = {
+		labels: [
+			"Внезапная покупка", "Стрижка", "Бассейн", "Школа", "Еда", 
+			"Плата жилья", "Ногти", "Бензин", "Дорога работа", "Юрист", 
+			"Детский сад", "Учебники", "Отпуск", "Театр", "Кредит", 
+			"Страховка", "Киберспорт", "Путешествия", "Кино", "Ипотека"
+		],
+		datasets: [
+			{
+				label: "Расходы",
+				data: [
+					1300.01, 3900.02, 3250.02, 1638.83, 2652.06, 
+					15271.09, 390.0, 975.56, 1340.79, 9110.05, 
+					16192.09, 2600.01, 6437.57, 1690.01, 26000.15, 
+					520.0, 520.0, 520.0, 520.0, 9586.33
+				].map(value => displayMode === "rub" ? value : (value / 130000 * 100).toFixed(2)),
+				backgroundColor: [
+					"#21703c", "#b48dad", "#bfc3a8", 
+					"#eed6cd", "#b83400", "#c88b79", 
+					"#ff9d73", "#90b89e", "#c7b2af", 
+					"#8891a8", "#edc915", "#f8ab9e", 
+					"#489177", "#8f9e3e", "#eef6c6", 
+					"#8f670a", "#c0a30f", "#afb9cf", 
+					"#c66a04", "#dbed15"
+				],
+				borderWidth: 0,
+			},
+		],
+	};
+
+	const displayData = data.labels.map((label, index) => ({
+		title: label,
+		value: data.datasets[0].data[index],
+		background: data.datasets[0].backgroundColor[index],
+	}));
+
 	return (
-		<>
-			<div>Analytics</div>
-		</>
+		<div className={styles.analyticsPageWrap}>
+			<div className={styles.analyticsPageContainer}>
+				<h1 className={styles.headerTitle}>Аналитика</h1>
+				{isEmptyPage? (
+					<div className={styles.analyticsBlankPage}>
+
+						<p className={styles.analyticsBlankPage__text}>К сожалению, этот раздел пока пуст. Начните вести учет финансов в приложении и сможет воспользоваться этим разделом.</p>
+						
+						<MoneyIcon classNames={styles.analyticsBlankPage__image}/>
+	
+					</div>
+				) : (
+					<div className={styles.analyticsPagesContent}>
+						<div className={styles.analyticsSelectContainer}>
+							<div className={styles.analyticsSelectOperation}>
+								<Select
+									name={"expenses"}
+									label={"Операции"}
+									options={["Расходы", "Доходы", "Анализ доходов и расходов"]}
+								/>
+							</div>
+							<div className={styles.analyticsSelectDateAndPeriod}>
+								<AppInput control={control} label={"Выбор даты"} type={InputTypeList.Date} name={"date"} />
+							</div>
+						</div>
+
+						<div className={styles.analyticsDisplayWrapper}>
+							<p className={styles.analyticsDisplay_title}>Отображать в:</p>
+							<div className={styles.analyticsDisplay}>
+								<input
+									name="value"
+									type="radio"
```

Comment:
can you replace plain input type definition by InputTypeList here and in other places?